### PR TITLE
Solving issue #33

### LINKED ISSUE #33: Update email feature to list contents of job search in email body
## Current Behaviour
Job search results is sent as a file attachment to the email.

## Changes Requested
Update to send the job search results in the email body.

### PR DESCRIPTION
**Pull request by feriodismo**
### 2. Description
Now the scraped info from website is send in the email body via **plain text,**

**Note:** it also can be send via HTML format for include styles, etc

Also I write a line (is commented) for other way to send secure email,

and I write a suggestion for parameterize the smtp port. 
### 3. Fill in checklist by marking [x]

- [x ] I've read the CONTRIBUTING.md
- [x ] I was assigned to this issue
- [x ] My code is formatted using required linting
- [x ] I've run my code locally and checked it works

### CODE REVIEW

File: job-search-web-scraping.py [hunk 1]
```diff
@@ -31,14 +33,19 @@ def send_email(sender_email_address, email_password,
...
+    # attach the text file
     part = MIMEApplication(open(file_path).read())
     part.add_header('Content-Disposition', 'attachment; filename="%s"' % basename(file_path))
     message.attach(part)
-
-    # set smtp server and port
+    # set smtp server and port (SUGGEST TO ENABLE THE PORT VIA CONFIG (PARAMETER))
     server = smtplib.SMTP(email_smtp, '587')
+
+    # server = smtplib.SMTP_SSL(email_smtp, '587') THIS OTHER WAY TO SEND SECURE EMAILS COMMENTING LINES 50-52-53
```
Comment (funbeedev) [inline job-search-web-scraping.py:47]: @feriodismo I checked this suggestion but didn't work for me. Does it work for you? 
I get the error:
```
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)
```

Comment (feriodismo) [reply job-search-web-scraping.py:47]: I got this error when I use port 587:

`
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1091)

`

If I use port 465 I got no error. ( I am using "smtp.gmail.com" )

Comment (feriodismo) [reply job-search-web-scraping.py:47]: Are you still getting the error changing the port?

Comment (funbeedev) [reply job-search-web-scraping.py:47]: @feriodismo It still doesn't work if I use port 465. If I do so the program will hang at the line:
```
server = smtplib.SMTP_SSL(email_smtp, '465')
```
It seems we are using different versions of the ssl module...
Log from my module:
```
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1123)
```
Versus yours:
```
ssl.SSLError: [SSL: WRONG_VERSION_NUMBER] wrong version number (_ssl.c:1091)
```
What's your python version? Mine is Python 3.8.5.

Comment (feriodismo) [reply job-search-web-scraping.py:47]: Maybe is that, I'm using Python 3.7.8 on windows

Comment (funbeedev) [reply job-search-web-scraping.py:47]: @feriodismo Okay, i'll remove that suggestion from the code and merge. Will need to be sure others can run it to include it in the code.
Other than that emailing works fine. Thanks for your contribution!

**Pull request merged**